source/data-api/authenticate.txt

```diff
@@ -0,0 +1,160 @@
+.. _data-api-authenticate-requests:
+
+==============================
+Authenticate Data API Requests
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Data API endpoints generally require that incoming requests include
+authentication information for the user calling the endpoint. This lets
+the endpoint enforce rules and validate document schemas for each
+request.
+
+Users authenticate requests by providing authentication information in
+the request headers. They can either use an authentication access token
+or directly include login credentials from any of the following
+authentication providers:
+
+- :ref:`Email/Password <email-password-authentication>`
+
+- :ref:`API Key <api-key-authentication>`
+
+- :ref:`Custom JWT <custom-jwt-authentication>`
```

Comment:
would bearer auth work with one of the other auth providers?

easiest way to test would prob be w custom function auth.

Reply:
Yeah it'll work with any auth provider - I can clarify